docs/codeql/codeql-cli/creating-and-working-with-codeql-packs.rst

```diff
@@ -34,24 +34,26 @@ You must specify:
 
 The ``codeql pack init`` command creates the directory structure and configuration files for a CodeQL pack. By default, the command creates a query pack. If you want to create a library pack, you must edit the ``qlpack.yml`` file to explicitly declare the file as a library pack by including the ``library:true`` property.
 
-Modifying an existing QL pack to create a CodeQL pack
------------------------------------------------------
+Modifying an existing legacy CodeQL pack to create a CodeQL pack
+----------------------------------------------------------------
+
 If you already have a ``qlpack.yml`` file, you can edit it manually to convert it into a CodeQL pack.
 
 #. Edit the ``name`` property so that it matches the format ``<scope>/<name>``, where ``<scope>`` is the name of the GitHub organization or user account that you will publish to.
 #. In the ``qlpack.yml`` file, include a ``version`` property with a semver identifier, as well as an optional ``dependencies`` block.
+#. Migrate the list of dependencies in ``libraryPathDependencies`` to the ``dependencies`` block. Specify the version range for each dependency. If the range is unimportant, or you are unsure of compatibility, you can specify ``"*"``, which indicates that any version is acceptable and will default to the latest version when ``codeql pack install`` is run.
 
 For more information about the properties, see ":ref:`About CodeQL packs <about-codeql-packs>`."
 
 Adding and installing dependencies to a CodeQL pack
 ---------------------------------------------------
-You can add dependencies on CodeQL packs using the command ``codeql pack add``. You must specify the scope, name, and version range.
+You can add dependencies on CodeQL packs using the command ``codeql pack add``. You must specify the scope, name, and (optionally) a compatible version range.
 
 ::
 
   codeql pack add <scope>/<name>@x.x.x <scope>/<other-name>
 
-The version range is optional. If you leave off the version range, the latest version will be added. Otherwise, the latest version that satisfies the requested range will be added.
+If you leave off the version range, the latest version will be added. Otherwise, the latest version that satisfies the requested range will be added.
```

Comment:
```suggestion
If you don't specify a version range, the latest version will be added. Otherwise, the latest version that satisfies the requested range will be added.
```

Reply:
Forgot to make this change. Will push it up shortly.